### DISCUSSION TITLE
Capstone DA - Flask Dashboard UI - Bagaimana cara save plot figure dari 2 plot yg berbeda pada 1 .png file

### DISCUSSION THREAD

**Discussion by MelissaRusli** [teamalgoritma/community#757] [Q&A] [answered]
Saya sedang di tahapan Task terakhir untuk membuat plot sendiri. 

Disini saya mencoba membuat 2 bar plot untuk menunjukan komparasi dari 5 ranking tertinggi berdasarkan frekuensi kategori applikasi free vs. paid
 
<img width="850" alt="Screen Shot 2023-09-02 at 3 14 14 AM" src="https://github.com/teamalgoritma/community/assets/141122159/eaf4a947-faad-4570-a4bb-11cca1065be2">

Bila saya melakukan plt.savefig dibagian akhir, hanya plot kedua yang akan tersimpan. Sehingga saat ini saya harus melakukan plt.savefig setelah code barplot pertama, kemudian juga pada setelah code barplot kedua. Sehingga pada saat ini saya mempunyai 2 file .png... supaya dapat menghasilkan plot sbb:

<img width="845" alt="Screen Shot 2023-09-02 at 3 14 35 AM" src="https://github.com/teamalgoritma/community/assets/141122159/e22005a0-7f8d-4133-a54d-3fa7c64d8f0d">

Mohon advise nya bagaimana cara untuk supaya kedua plot tersebut tersimpan pada satu file .png yang sama?

**Comment by arkanayudhistira** [comment, marked as answer]
Dear @MelissaRusli,

Untuk memasukkan beberapa plot dalam 1 gambar, ibu dapat memanfaatkan `subplot` seperti yang sudah ibu gunakan sekarang, namun untuk menempatan code nya kurang sesuai pada ax nya, silahkan mengacu ke referensi berikut ini ibu untuk membuat `subplot` pada `matplotlib.pyplot` : 
https://matplotlib.org/stable/gallery/subplots_axes_and_figures/subplots_demo.html

Semoga jawaban saya membantu!

**Reply by MelissaRusli** [reply]
Terima Kasih. Saya sudah berhasil untuk resolve ini.
